### PR TITLE
Tiny doc typo in kernel_computed_fields

### PR DESCRIPTION
**Pull request by christophernhill**
Changes V to v in doc to fix a typo - and try a PR!

### CODE REVIEW

**Review by navidcy — APPROVED** [review]
This looks great. :)

**Review by ali-ramadhan — APPROVED** [review]
Your first PR since PR #1 haha!

**Comment by ali-ramadhan** [comment]
PS @christophernhill: Buildkite tests did not run on this PR since we disable Buildkite on PRs from forks (to avoid random PRs executing potentially malicious code on Tartarus and Sverdrup).

So we usually open and merge branches from the repo itself.

**Comment by christophernhill** [comment]
> PS @christophernhill: Buildkite tests did not run on this PR since we disable Buildkite on PRs from forks (to avoid random PRs executing potentially malicious code on Tartarus and Sverdrup).
> 
> So we usually open and merge branches from the repo itself.

OK - I'll put it in a branch sometime today

File: src/Fields/kernel_computed_field.jl [hunk 1]
```diff
@@ -31,7 +31,7 @@ struct KernelComputedField{X, Y, Z, S, A, G, K, F, P} <: AbstractField{X, Y, Z,
     u, v, w = model.velocities
     
     U = AveragedField(u, dims=(1, 2))
-    V = AveragedField(V, dims=(1, 2))
+    V = AveragedField(v, dims=(1, 2))
```
Comment (glwagner) [inline src/Fields/kernel_computed_field.jl:34]: 😸

**Comment by ali-ramadhan** [comment]
> OK - I'll put it in a branch sometime today

Ah I wouldn't worry about it for this PR, I'd just merge it. Was just a heads up for future PRs.

**Comment by ali-ramadhan** [comment]
Closing as this PR has been superseded by PR #1335.

**Pull request closed without merging**